_includes/header.html

```diff
@@ -4,7 +4,13 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-
+    <noscript>
+      <br>
+      <div style = "text-align: center;">
+      <p style="color: #eed202"> Please enable JavaScript to view the page</p>
```

Comment:
```suggestion
      <p style="color:  #EBC017"> Please enable JavaScript to view the page</p>
```
Consider using this color instead.

Reply:
Ok will do the changes asap